mods/ra/weapons/missiles.yaml

```diff
@@ -448,6 +308,7 @@ SubMissileAA:
 		ValidTargets: Air, Ground, Water
 
 SCUD:
+	Inherits: ^AntiGroundMissile
```

Comment:
SCUD uses `Bullet`, so except the (unintentional) contrail length change, these would have no effect.